src/core/messages.ts

```diff
@@ -39,6 +39,21 @@ const MessageActionsToEventsMap: Map<ChatMessageActions, MessageEvents> = new Ma
   [ChatMessageActions.MessageDelete, MessageEvents.Deleted],
 ]);
 
+/**
+ * The order in which results should be returned when performing a paginated query (e.g. message history).
+ */
+export enum ResultOrder {
```

Comment:
`OrderBy`?

Nitpick, not strong opinion.

Just thinking it might be easier to remember to type `messages.get({orderBy: OrderBy.NewestFirst})` rather than `messages.get({orderBy: ResultOrder.NewestFirst})`.

Another nitpick: should we define this in `chat-api.ts` instead?

Reply:
Worth bringing up with ecosystems (that's where `ResultOrder` originates from) - once the DR is agreed and we definitely do this, we can deffo consider this change :)

Reply:
Changed to OrderBy